test/audits/aria-on-reserved-element-test.js

```diff
@@ -0,0 +1,92 @@
+// Copyright 2014 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+(function() {
+    module('AriaOnReservedElement');
+    var rule = axs.AuditRules.getRule('ariaOnReservedElement');
+
+    test('Non-reserved element with role and aria- attributes', function() {
+        var fixture = document.getElementById('qunit-fixture');
+        var widget = fixture.appendChild(document.createElement('div'));
+        var expected = { result: axs.constants.AuditResult.NA };
+        widget.setAttribute('role', 'spinbutton');
+        widget.setAttribute('aria-hidden', 'false');  // global
+        widget.setAttribute('aria-required', 'true');  // supported
+        widget.setAttribute('aria-valuemax', '79');  // required
+        widget.setAttribute('aria-valuemin', '10');  // required
+        widget.setAttribute('aria-valuenow', '50');  // required
+        deepEqual(rule.run({ scope: fixture }), expected);
+    });
+
+    test('Non-reserved element with role only', function() {
+        var fixture = document.getElementById('qunit-fixture');
+        var widget = fixture.appendChild(document.createElement('div'));
+        var expected = { result: axs.constants.AuditResult.NA };
+        widget.setAttribute('role', 'spinbutton');
+        deepEqual(rule.run({ scope: fixture }), expected);
+    });
+
+    test('Non-reserved element with aria-attributes only', function() {
+        var fixture = document.getElementById('qunit-fixture');
+        var widget = fixture.appendChild(document.createElement('div'));
+        var expected = { result: axs.constants.AuditResult.NA };
+        widget.setAttribute('aria-hidden', 'false');  // global
+        deepEqual(rule.run({ scope: fixture }), expected);
+    });
+
+    test('Reserved element with role and aria- attributes', function() {
+        var fixture = document.getElementById('qunit-fixture');
+        var widget = fixture.appendChild(document.createElement('meta'));
+        var expected = { elements: [widget], result: axs.constants.AuditResult.FAIL };
```

Comment:
Could you move the definition of  `expected` down to the line before it's used, and add a brief explanation for each one? e.g.
// Reserved elements can't take any ARIA attributes.